v0/destinations/freshmarketer/data/FRESHMARKETERIdentifyConfig.json

```diff
@@ -0,0 +1,194 @@
+    ],
+    "required": false
+  },
+  {
+    "destKey": "state",
+    "sourceKeys": [
+      "traits.state",
+      "traits.address.state",
+      "context.traits.state",
+      "context.traits.address.state"
+    ],
+    "required": false
+  },
+  {
+    "destKey": "country",
+    "sourceKeys": [
+      "traits.address.country",
+      "traits.country",
+      "context.traits.address.country",
+      "context.traits.country"
+    ],
+    "required": false
+  },
+  {
+    "destKey": "zipcode",
+    "sourceKeys": "zipcode",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "sales_accounts",
+    "sourceKeys": [
+      "traits.salesAccounts",
+      "traits.sales_accounts",
+      "context.traits.salesAccounts",
+      "context.traits.sales_accounts"
+    ],
+    "required": false
+  },
+  {
+    "destKey": "territory_id",
+    "sourceKeys": ["traits.territoryId", "context.traits.territoryId"],
+    "required": false
+  },
+  {
+    "destKey": "lead_source_id",
+    "sourceKeys": ["traits.leadSourceId", "context.traits.leadSourceId"],
+    "required": false
+  },
+  {
+    "destKey": "owner_id",
+    "sourceKeys": ["traits.ownerId", "context.traits.ownerId"],
+    "required": false
+  },
+  {
+    "destKey": "subscription_types",
+    "sourceKeys": [
+      "traits.subscriptionTypes",
+      "context.traits.subscriptionTypes"
+    ],
+    "required": false
+  },
+  {
+    "destKey": "medium",
+    "sourceKeys": ["traits.medium", "context.traits.medium"],
+    "required": false
+  },
+  {
+    "destKey": "campaign_id",
+    "sourceKeys": [
+      "traits.campaignId",
+      "traits.campaign_id",
+      "context.traits.campaignId",
+      "context.traits.campaign_id"
+    ],
+    "required": false
+  },
+  {
+    "destKey": "keyword",
+    "sourceKeys": ["traits.keyword", "context.traits.keyword"],
+    "required": false
+  },
+  {
+    "destKey": "time_zone",
+    "sourceKeys": ["traits.timeZone", "context.traits.timeZone"],
+    "required": false
+  },
+  {
+    "destKey": "facebook",
+    "sourceKeys": ["traits.facebook", "context.traits.facebook"],
+    "required": false
+  },
+  {
+    "destKey": "twitter",
+    "sourceKeys": ["traits.twitter", "context.traits.twitter"],
+    "required": false
+  },
+  {
+    "destKey": "linkedin",
+    "sourceKeys": ["traits.linkedin", "context.traits.linkedin"],
+    "required": false
+  },
+  {
+    "destKey": "created_at",
+    "sourceKeys": "createdAt",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "updated_at",
+    "sourceKeys": "timestamp",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "contact_status_id",
+    "sourceKeys": ["traits.contactStatusId", "context.traits.contactStatusId"],
+    "required": false
+  },
+  {
+    "destKey": "lifecycle_stage_id",
+    "sourceKeys": [
+      "traits.lifecycleStageId",
+      "context.traits.lifecycleStageId"
+    ],
+    "required": false
+  }
+]
```

Comment:
All of the fields are provided with datatypes. What happens if we provide another datatype? Let's say lifecycle_stage_id expects a number what happens if we provide any string or boolean?

Reply:
Yeah, we need to check data types, will update it.

Reply:
Done